start_server.sh

```diff
@@ -1 +1 @@
-gunicorn -w 4 -b 0.0.0.0:5000 app:app
+gunicorn -w 4 -t 60 -b 0.0.0.0:5000 app:app
```

Comment:
what's this for?

Reply:
peep the pr comment. its to prevent timeout when we query yelp

Reply:
because when we have 4 things pinging yelp with the same api key, we burn through our queryies/second 4x as fast so we reduce the ping rate accordingly.